exchanges/bybit/bybit_wrapper.go

```diff
@@ -96,6 +96,9 @@ func (e *Exchange) SetDefaults() {
 				currency.NewCode("SHIB1000"):        currency.SHIB,
 			},
 		),
+		TradingRequirements: protocol.TradingRequirements{
+			SpotMarketOrderAmountPurchaseQuotationOnly: true,
```

Comment:
👀  Erm. Okay then.
🥃 This feels like `Quotation: true` 😂

Reply:
Can you check this change out: https://github.com/thrasher-corp/gocryptotrader/pull/1672/commits/f583c04a131b4ed8366803ffa1900d50ea5f0595

Reply:
Smells good 👍